src/views/Create.vue

```diff
@@ -198,7 +198,7 @@ watch(selectedNetworkId, () => {
           ref="pagesRefs"
           :key="page.id"
           :disabled="!accessiblePages[page.id]"
-          class="px-3 py-1 block lg:w-full rounded text-left scroll-mr-3 first:ml-auto last:mr-auto"
+          class="px-3 py-1 block lg:w-full rounded text-left scroll-mr-3 first:ml-auto last:mr-auto whitespace-nowrap"
```

Comment:
Added whitespace-nowrap because of this
![image](https://user-images.githubusercontent.com/97898172/230349036-33b13be9-da7f-47bf-b133-61179ab3124f.png)